README.md

```diff
@@ -51,29 +48,43 @@ npm install
 ember serve
 ```
 
-Visit [http://localhost:4200](http://localhost:4200)
+Afterwards, visit [http://localhost:4200](http://localhost:4200) on your browser.
+
+Note: On Mac, if you get the error `Error: EMFILE: too many open files, watch`, try installing Watchman. Install [Homebrew](https://brew.sh/) if you don't have it. Then, in your terminal, run `brew install watchman`.
 
-Note: If you get an error on macOS mentioning `"Error: EMFILE: too many open files, watch"`, try installing Watchman. Install [Homebrew](https://brew.sh/) if you don't have it. Then in you terminal `brew install watchman`.
 
 ## Running tests
 
-Use `npm` to run tests instead of `ember`, since we have additional
-tests like spellchecking that are not part of The Ember app's tests.
+Use `npm test` to run tests locally. In addition to the Ember app, we check for broken links.
 
-```
-npm install
+```bash
 npm test
 ```
 
 ### Linting and spellchecking
 
-The guides are spellchecked and linted for Markdown consistency. You can test your contributions by running `npm run lint:md`. Linting and spellchecking must pass or they will fail in Travis-CI.  See [CONTRIBUTING.md](CONTRIBUTING.md) for more information on linting and spellchecking.
```

Comment:
I think this is relevant info about where to find the info on the lint and spellchecking.

Reply:
Sounds good. I added the last 2 paragraphs back.